data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessorConfig.java

```diff
@@ -146,6 +194,10 @@ public class KeyValueProcessorConfig {
             "In this case, strict means that whitespace is trimmed and lenient means it is retained in the key name and in the value. " +
             "Default is <code>lenient</code>.")
     @NotNull
+    @ExampleValues({
```

Comment:
Let's avoid sample values on enumerations. Those are already available in the enum list.